src/flexmeasures_client/client.py

```diff
@@ -685,6 +703,19 @@ async def update_asset(self, asset_id: int, updates: dict) -> dict:
             )
         return updated_asset
 
+    async def delete_asset(self, asset_id: int):
+        """Deletes an asset and its sensors & data.
+
+        This function raises a ValueError when an unhandled status code is returned.
+        """
+        answer = input(f"Delete asset {asset_id} and all its sensors and data? [yN]")
+        if answer.lower() not in ["y", "yes"]:
+            print("Aborting ...")
+            return {}
```

Comment:
Perhaps there should be a way for scripts to bypass these prompts, for automation purposes?

Reply:
Yes, I'll add a `force=True` option.